upload-to-ecr/action.yml

```diff
@@ -30,18 +26,25 @@ inputs:
   ecr-repository:
     required: true
     description: ECR Repository for image upload
-  aws-kms-key:
+  env-vars:
+    required: false
+    description: Environment Variables
+    default: "false"
+  github-token-parameter-store-path:
     required: true
-    description: KMS Key to sign image with
-  vuln-severity-cutoff:
+    description: Path in Parameter Store for Github Token
+  image-tag:
     required: false
-    default: critical
-    description: Severity cutoff for Anchore Scan
+    description: Tag for image
+    default: ${{ github.sha }}
```

Comment:
Turns out this doesn't actually work.

I can either make this input required, or have a step that has an output value of the `image-tag` if it's provided or the SHA if it's not.

Reply:
Oh, I see, you want the option to provide a tag and not hard-code it to the SHA. We could tag it both with the SHA and also with another user-provided tag, and only perform that step if it was provided?

Reply:
I like that approach - I will update this.

Reply:
Will the additional tag need to be used for the signing / SBOM steps, or can those continue to just use the SHA?

Reply:
I _believe_ that in ECR, the tags are all on the same image, as well as the signature and the SBOM, so it shouldnt be necessary to replicate the steps.

Reply:
Added another step that only runs when the `additional-image-tag` input is provided. Duplicating it doesn't seem ideal, but I wasn't sure of another way to conditionally add the tag.

Confirmed this works (when providing an additional tag as well as omitting it) in one of my team's repos.